.github/dependabot.yml

```diff
@@ -7,3 +7,38 @@ updates:
     directory: "/"
     schedule:
       interval: "daily"
+
+  - package-ecosystem: pip
+    directory: /doc
+    schedule:
+      interval: daily
+
+  - package-ecosystem: pip
+    directory: /
+    schedule:
+      interval: daily
+
+  - package-ecosystem: cargo
+    directory: /test/language_data
+    schedule:
+      interval: daily
+
+  - package-ecosystem: bundler
+    directory: /test/language_data
+    schedule:
+      interval: daily
+
+  - package-ecosystem: gomod
+    directory: /test/language_data
+    schedule:
+      interval: daily
+
+  - package-ecosystem: pip
+    directory: /test/language_data
+    schedule:
+      interval: daily
+
+  - package-ecosystem: maven
+    directory: /test/language_data
+    schedule:
+      interval: daily
```

Comment:
```suggestion
```

Removing scans on test data (vulnerable code is never installed, but we need those package lists for testing our own scanner.)